package.json

```diff
@@ -8,7 +8,7 @@
     "start:node": "cd api && npm run watch",
     "test:node": "cd api && npm run test",
     "test:node:watch": "cd api && npm run test:watch",
-    "test:coverage:ng": "ng test --code-coverage --watch",
+    "test:coverage:ng": "ng test rpa-pui-manager --code-coverage --watch",
```

Comment:
Don't do covergage like this, put the config in angular.json so its part of ng test.
The reason for this is that the pipeline runs the test command only and expects coverage reports to be generated like that. See JUI for example

Reply:
I've added `"codeCoverage": true,` to the angular.json file now, but I won't be removing it from here as the task name does indicates code coverage.